sdk/dotnet/IntegrationMetric.cs

```diff
@@ -151,7 +96,13 @@ public partial class IntegrationMetric : Pulumi.CustomResource
         public Output<string?> Tags { get; private set; } = null!;
 
         /// <summary>
-        /// (optional) whitelist using regular expression
+        /// (optional) allowlist using regular expression
+        /// </summary>
+        [Output("vhostAllowlist")]
+        public Output<string?> VhostAllowlist { get; private set; } = null!;
+
+        /// <summary>
+        /// **Deprecated**
         /// </summary>
         [Output("vhostWhitelist")]
```

Comment:
Should this have a lang-specific deprecation notice (go, node and python do)

Reply:
CSharp doesn't pick up the deprecations like that - that's only a command so it's fine :)